### PR TITLE
Add Google Analytics tracking

### PR DESCRIPTION
**Pull request by martinhoaragao**

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -76,4 +76,8 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  # Google Analytics
+  GA.tracker = "UA-67237593-1"
+
```
Comment (frmendes) [inline config/environments/production.rb]: Unnecessary newline.

File: config/environments/production.rb [hunk 2]
```diff
@@ -76,4 +76,8 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  # Google Analytics
+  GA.tracker = "UA-67237593-1"
```
Comment (frmendes) [inline config/environments/production.rb]: Can we move this to an env variable?

**Comment by joaofcosta** [comment]
@martinhoaragao How's this going? If you do not have time or patience to finish I could take over if you wish :)

**Comment by martinhoaragao** [comment]
Sorry @joaofcosta! I will pick this up asap.

**Comment by martinhoaragao** [comment]
Can you check again please @fribmendes ?

**Pull request merged**